src/app/lib/analyticsUtils/index.js

```diff
@@ -127,20 +126,12 @@ export const getReferrer = (platform, origin, previousPath) => {
 };
 
 export const getAtUserId = () => {
-  if (!onClient()) return null;
```

Comment:
Doesn't this seem to undo the work I did [here](https://github.com/bbc/simorgh/pull/4812)?

Reply:
I think once you merge in changes from `latest` you might be good.